lib/bootstrap/patches/gems.rb

```diff
@@ -29,3 +29,16 @@ def get(path, data = {}, content_type = 'application/x-www-form-urlencoded', req
     end
   end
 end
+
+module ::Bundler
```

Comment:
Instead of patching this, i think we were just sending an object instead of a string. Modified to send along a string instead of a `Gem::Platform` instance. 

```
`Caused by: org.jruby.exceptions.NoMethodError: (NoMethodError) undefined method `empty?' for #<Gem::Platform:0x637b58c9>`
```